src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        final int arrLen = 6;
+
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[arrLen];
+        for (int i = 0; i < arrLen; i++) {
```

Comment:
for more consistency
```suggestion
        for (int i = 0; i < figures.lenght; i++) {
```